docker/scripts/run-mx-ngraph-validation-test.sh

```diff
@@ -0,0 +1,189 @@
+
+
+if [ ! -z "${MX_NG_MODEL}" ] ; then
+    ng_mx_model="${MX_NG_MODEL}"
+fi
+if [ ! -z "${1}" ] ; then
+    ng_mx_model="${1}"
+fi
+if [ -z "${ng_mx_model}" ] ; then
+    ( >&2 echo "SYNTAX ERROR: First and only parameter should be ng_mx_model." )
+    ( >&2 echo "Supported ng_mx_model combinations are:")
+    ( >&2 echo "    mlp-mnist  resnet110-cifar10")
+    exit 1
+fi
+
+set -e  # Make sure we exit on any command that returns non-zero
+set -u  # No unset variables
+set -o pipefail # Make sure cmds in pipe that are non-zero also fail immediately
+
+# ===== run_MLP_MNIST() ========== 
+# Function to run the example/image-classification/train_mnist.py
+# Note:  read_data() will automatic download data from http://yann.lecun.com/exdb/mnist/ (train-images-idx3-ubyte.gz, t10k-images-idx3-ubyte.gz)
+run_MLP_MNIST() {
+    # Make sure the bash shell prompt variables are set, as virtualenv crashes
+    # if PS2 is not set.
+    PS1='prompt> '
+    PS2='prompt-more> '
+    virtualenv -p "${PYTHON_BIN_PATH}" "${venv_dir}"
+    source "${venv_dir}/bin/activate"
+    cd python && pip install -e . && cd ../
+    xtime="$(date)"
+    echo  ' '
+    echo  "===== Running Ngraph Mxnet Daily Validation on CPU-Backend at ${xtime} ====="
+    echo  ' '
+    echo  "===== PWD is $PWD ====="
+    # In train_mnist.py script, OMP_NUM_THREADS (omp_max_thread) and KMP_AFFINITY are explicitly
+    # set only for the nGraph run by default. NUM_EPOCHS = 20 
+    # Test parameters
+    export TEST_MLP_MNIST_LOG_DIR="${HOME}/ng-mx"
+    # Run the test
+    python example/image-classification/train_mnist.py 
+    echo "===== Daily Validation CPU-Backend Pipeline Exited with $? ====="
+
+}  # run_MLP_MNIST()
+
+# ===== run_RESNET110_CIFAR10() ========== 
+# Function to run the example/image-classification/train_cifar10.py
+# Note: download_cifar10() will automatic download data from http://data.mxnet.io/data/cifar10
+
+run_RESNET110_CIFAR10() {
+    # Make sure the bash shell prompt variables are set, as virtualenv crashes
+    # if PS2 is not set.
+    PS1='prompt> '
+    PS2='prompt-more> '
+    virtualenv -p "${PYTHON_BIN_PATH}" "${venv_dir}"
+    source "${venv_dir}/bin/activate"
+    cd python && pip install -e . && cd ../
+    xtime="$(date)"
+    echo  ' '
+    echo  "===== Running Ngraph Mxnet Daily Validation on CPU-Backend at ${xtime} ====="
+    echo  ' '
+    echo  "===== PWD is $PWD ====="
+    # In train_cifar10.py script, OMP_NUM_THREADS (omp_max_thread) and KMP_AFFINITY are explicitly
+    # set only for the nGraph run.  Thus, they are not set here.
+    # Test parameters
+    export TEST_RESNET110_CIFAR10_LOG_DIR="${HOME}/ng-mx"
+    export TEST_RESNET110_CIFAR10_EPOCHS="${MX_NG_EPOCHS:-}"
+    if [ -z "${TEST_RESNET110_CIFAR10_EPOCHS}" ] ; then
+        export TEST_RESNET110_CIFAR10_EPOCHS=1  # Default is 300 epoches
+    fi
+    # Run the test
+    python example/image-classification/train_cifar10.py --num-epochs ${TEST_RESNET110_CIFAR10_EPOCHS}
```

Comment:
This is functional testing. I would like to all default parameters, but only run 1 epochs for resnet_cifar10.